.github/workflows/workflow-repo-standards-validation.yml

```diff
@@ -0,0 +1,120 @@
+name: Reusable workflow to validate organization standards
+
+on:
+  workflow_call:
+
+jobs:
+  validate-name:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check repository name
+      run: |
+        repo_name="${{ github.repository }}"
+        lower_repo_name="${repo_name,,}"
+        
+        if [[ "$repo_name" != "$lower_repo_name" ]]; then
+          echo "The repository name '$repo_name' should contain only lowercase letters." >&2
+          exit 1
+        elif [[ "$repo_name" == *"_"* ]]; then
+          echo "The repository name '$repo_name' should contain dashs ( - ) and not underscores ( _ )." >&2
+          exit 1
+        fi
+      if: always()
+      shell: bash
+
+    - name: Check repository visibility
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        repo_info=$(curl -H "Authorization: token $GITHUB_TOKEN" -s "https://api.github.com/repos/${{ github.repository }}")
+        repo_visibility=$(echo $repo_info | jq -r .private)
+        
+        if [[ "$repo_visibility" == "true" ]]; then
+          echo "The repository '${{ github.repository }}' is private and needs to be public". >&2
+          exit 1
+        fi
+      if: always()
+      shell: bash
+
+    - name: Check repository description
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        repo_info=$(curl -H "Authorization: token $GITHUB_TOKEN" -s "https://api.github.com/repos/${{ github.repository }}")
+        repo_description=$(echo $repo_info | jq -r .description)
+        
+        if [[ "$repo_description" == "null" || -z "$repo_description" ]]; then
+          echo "The repository does not have a description, please add one." >&2
+          exit 1
+        fi
+      if: always()
+      shell: bash
+
+    - name: Check branch protection
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        protection=$(curl -H "Authorization: token $GITHUB_TOKEN" -s "https://api.github.com/repos/${{ github.repository }}/branches/main/protection")
+        message=$(echo $protection | jq -r .message)
+
+        echo $protection | jq -r .message
+        
+        if [[ "$message" == "Branch not protected" ]]; then
+          echo "Error: The main branch is not protected!" >&2
+          exit 1
+        fi
+      if: always()
+      shell: bash
+
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Check for README.md
+      run: |
+        FILE="README.md"
+        if [[ ! -f "$FILE" ]]; then
+          echo "Error: '$FILE' does not exist in the repository, please create one." >&2
+          exit 1
+        fi
+      if: always()
+      shell: bash
+
+    - name: Check for LICENSE file
+      run: |
+        FILE="LICENSE"
```

Comment:
LICENSE.md

Reply:
I wasn't sure either but when I looked at the org's repo they all said LICENSE and not LICENSE.md but that doesn't happen for the readme
<img width="1152" alt="Screenshot at Oct 07 13-42-54" src="https://github.com/ai-cfia/github-workflows/assets/9827730/33d80a18-eeb5-49f0-aa69-d3476de8980c">

Reply:
I used Octokit to verify the presence of LICENSE and it checks the repo's details not the actual file so it should cover all possible names for it
LICENSE, LICENSE.md or LICENSE.txt